### PR TITLE
update onboarding docs

### PR DESCRIPTION
**Pull request by RayBB**
Ironically, no associated ticket. I'm just trying to improve our onboarding docs before adding anything about remote development (gitpods / github codespaces).

The improvements are generally:
* Putting linebreaks in long lines. (this has no visual changes)
* Wordsmithing - making descriptions shorter and hopefully clearer
* Removing links that aren't useful
* Add the docker command to the main readme

I try to err on the side of being as clear as possible for new users.

If you think anything should be phrased differently feel free to edit the pr directly, make a comment, or make a PR.

### Stakeholders

@jamesachamp

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,28 +24,23 @@ A deep dive into the technical details, architecture, and code structure behind
 
 ### Code of Conduct
 
-Before continuing, please familiarize yourself with our [code of conduct](https://github.com/internetarchive/openlibrary/blob/master/CODE_OF_CONDUCT.md). We are a non-profit, open-source, inclusive project, and we believe everyone deserves a safe place to make the world a little better. We're committed to creating this safe place:
-
-https://github.com/internetarchive/openlibrary/blob/master/CODE_OF_CONDUCT.md
+Before continuing, please familiarize yourself with our [code of conduct](https://github.com/internetarchive/openlibrary/blob/master/CODE_OF_CONDUCT.md). 
+We are a non-profit, open-source, inclusive project, and we believe everyone deserves a safe place to make the world a little better. We're committed to creating this safe place:
```
Comment (jamesachamp) [inline CONTRIBUTING.md:28]:
```suggestion
We are a non-profit, open-source, inclusive project, and we believe everyone deserves a safe place to make the world a little better. We're committed to creating this safe place.
```

File: Readme.md [hunk 2]
```diff
@@ -34,7 +34,10 @@ Here's a quick public tour of Open Library to get your familiar with the service
 
 ## Installation
 
-The development environment can be set up using the [Docker Instructions](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md). You can also watch the [video tutorial](https://archive.org/embed/openlibrary-developer-docs/openlibrary-docker-set-up.mp4) for a more detailed explanation.
+Run `docker-compose up` and visit http://localhost:8080
```
Comment (jamesachamp) [inline Readme.md:37]: Do you think that we should explicitly mention that Docker Compose needs to be installed before running this command?

**Review by jamesachamp — CHANGES_REQUESTED** [review]
I think that this will be good to go once the colon is replaced by a period in the Code of Conduct section.

The Installation section is probably fine.  I'm assuming that folks will read the Docker readme if they encounter any issues.

File: Readme.md (hunk 2 above)
Comment (RayBB) [reply Readme.md:37]: I think that's a fair question. I think I've seen plenty of projects not mention it and plenty mention it. I plan to clean up the docs further, especially as we add gitpod. I like to keep the main readme simple and link out to the more detailed docs but that's just me.

Comment (jimchamp) [reply Readme.md:37]: Keeping the project readme simple seems reasonable to me.

**Review by jimchamp — APPROVED** [review]
Lgtm.  Thanks for all of the work that you're doing to improve Open Library's documentation!

**Pull request merged**